prow/cluster/jobs/istio/proxy/istio.proxy.master.yaml

```diff
@@ -84,4 +84,24 @@ postsubmits:
       preset-service-account: "true"
       preset-bazel: "true"
     spec:
-      <<: *bazel_spec
+      containers:
+      - image: gcr.io/istio-testing/prowbazel:0.4.10
+        args:
+        - "--repo=github.com/$(REPO_OWNER)/$(REPO_NAME)=$(PULL_REFS)"
+        - "--clean"
+        - "--timeout=60"
```

Comment:
Piotr, this is the key line. Increase timeout to 60 minutes

Reply:
Any reason not to globally bump `*bazel_spec` to 60mins?

Reply:
not entirely sure. will have to talk to Seb

Reply:
I second this approach. Shorter jobs are not affacted (run time unchanged) and super long jobs that exceeded 60 should have been killed anyway. Much cleaner solution than duplicating the entire spec just to change one line